z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-pre-check/tasks/main.yaml

```diff
@@ -69,11 +54,6 @@
   shell: openstack catalog show -f json cinder | jq -r '.endpoints[] | "\(.url)"' | head -n 1
   register: storage_provider_url
 
-- name: Check icic resource requirements
```

Comment:
Why not checking ICIC resource?

Reply:
ICIC service and ICIC resource check moves into README doc, our code does not need to judge it because some ICIC errors like `not enough memory` will not affect ansible running, but it will let our ansible skip.

Reply:
OK, but it check whether ICIC resource meet OCP requirement, I think it 's better to do it or say it in README.md